src/app/components/data-entry-grid/index.html

```diff
@@ -233,6 +238,13 @@ <h2>Passing cell editor parameters</h2>
         You can pass inputs to the SKY UX components used in cell editors via the <a href="https://www.ag-grid.com/javascript-grid-column-properties/">column definition's <sky-code>cellEditorParams</sky-code> property.</a> The component properties are nested under <sky-code>skyComponentProperties</sky-code>. All component properties should be supported, and you can see the defined types by importing <sky-code>SkyCellEditorDatepickerProperties</sky-code> or <sky-code>SkyCellEditorAutocompleteProperties</sky-code>. <sky-code>cellEditorParams</sky-code> can also be a function that returns a param object for dynamic editing params. See the demo for examples of using an object or a function.
       </p>
 
+      <h2>Using other SKY UX components in columns</h2>
+      <p>
+        If you need full control over the SKY UX component rendered in a cell, you can create your own <a href="https://www.ag-grid.com/javascript-grid-cell-rendering-components/">cell renderer</a> and place the SKY UX component inside of it. Here are some cases where you will want to do this:
```

Comment:
Can we change `If you need` to `For`? just to tighten up a little? Also, change `the` to `a` in front of `SKY UX`. I'd also delete `you can` to make this sound less conditional. And I'd delete the second instance of `SKY UX` to tighten up and avoid the repetition. And I'd delete 'of` after `inside` because it's just an extra word.

I'd also delete the "Here are some cases...` sentence as Denise suggested. See my note below for more info on that.